infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java

```diff
@@ -514,7 +514,8 @@ private AzureCustomizableParameter getDefaultNodeParameters() {
                                               additionalProperties);
     }
 
-    private void deployInstancesWithNodes(int nbInstancesToDeploy, AzureCustomizableParameter params) {
+    private void deployInstancesWithNodes(int nbInstancesToDeploy, boolean reuseCreatedInstances,
```

Comment:
I would name this new parameter forceNewInstancesDeployment and use it with if (forceNewInstancesDeployment || expectInstancesAlreadyCreated(false, true))

Reply:
ok .. but reuseCreatedInstances is also used in aws infra class